test/01LessonOneTest.js

```diff
@@ -5,11 +5,12 @@ var assert = require('assert');
 
 function returnTrue() {
     // should return true
-
+    return true;
 }
 
 function returnFalse() {
     // should return false
+    return false;
 
 }
 
```

Comment:
Guess what Paul, you caint spell!!!
* one
* two

[link text](https://media.giphy.com/media/j2rCxv6luaLq8/giphy.gif)

![link text](https://media.giphy.com/media/j2rCxv6luaLq8/giphy.gif)